core/trino-main/src/main/java/io/trino/type/DoubleOperators.java

```diff
@@ -55,7 +57,7 @@ public final class DoubleOperators
     private static final double MIN_BYTE_AS_DOUBLE = -0x1p7;
     private static final double MAX_BYTE_PLUS_ONE_AS_DOUBLE = 0x1p7;
 
-    private static final ThreadLocal<DecimalFormat> FORMAT = ThreadLocal.withInitial(() -> new DecimalFormat("0.0###################E0"));
+    private static final ThreadLocal<DecimalFormat> FORMAT = ThreadLocal.withInitial(() -> new DecimalFormat("0.0###################E0", new DecimalFormatSymbols(Locale.US)));
```

Comment:
use `Locale.ENGLISH` and import statically 
(ENGLISH is the locale we use by default to pin locale)